tests/libcxx/expected_results.txt

```diff
@@ -177,6 +177,9 @@ std/ranges/range.access/size.pass.cpp FAIL
 # libc++ doesn't implement P2652R2 "Disallowing User Specialization Of allocator_traits"
 std/utilities/memory/allocator.traits/allocate_at_least.pass.cpp FAIL
 
+# libc++ doesn't implement P2675R1 "Improving std::format's Width Estimation"
+std/utilities/format/format.functions/unicode.pass.cpp FAIL

```

Comment:
It seems this feature is already implemented in [v17](https://github.com/llvm/llvm-project/blob/a08b9fb9675e99dd44eef04fa48bd51bb340c491/libcxx/docs/ReleaseNotes/17.rst#:~:text=P2675R1%20%2D%20format%27s%20width%20estimation%20is%20too%20approximate%20and%20not%20forward%20compatible). Is the failure due to the test are using older version? Is the comment suitable here?

Reply:
Yes and yes, says @CaseyCarter. We periodically update our LLVM submodule so this expected failure will be cleared out at that time, but until then it must be listed here.